src/backend/distributed/commands/utility_hook.c

```diff
@@ -1315,7 +1318,10 @@ ExecuteDistributedDDLJob(DDLJob *ddlJob)
 
 		PG_TRY();
 		{
-			ExecuteUtilityTaskList(ddlJob->taskList, localExecutionSupported);
+			if (ddlJob->taskList)
+			{
```

Comment:
Maybe this check should be done inside ExecuteUtilityTaskList, i.e. it early exits if the list is empty.

Reply:
that sounds better to me

Reply:
done